doc/user_manual/tasks.md

```diff
@@ -0,0 +1,141 @@
+# Tasks
+
+To create a new task, you have to navigate to *Tasks > New Task*. You will get the following window in which you can create a new task. Some of the fields are mandotory, some others are filled with default values. 
+
+1. **Name**: provide a name for the task you want to create. This is how the task will be referenced with during the monitoring phase (see **link**) therefore it should be relatively explicit to facilitate its monitoring.
+
+2. **Hashlist**: select the hashlist you want to target in this specific task. Tasks are ordered by their IDs. Supertasks (**see ref to advanced usage**) are at the bottom of the list ordered by their respective IDs.
+
+3. **Command Line**: provide in this field the attack command that will be executed by the agent on the targeted hashlist using the selected binary (see below). Note that *#HL#* is filled in by default in the command line. It is a placeholder for the hashlist and will be replaced automatically at execution time by the agent with the correct path to the hashlist file. Therefore you should not remove it nor include the filename for the hashlist. If for example you want to perform a mask attack of 6 digits, the command line would look like ```#HL# -a3 ?d?d?d?d?d?d```.
+In case you want to perform a dictionary attack with rules, you have to select the corresponding files in the right table. If it is a wordlist, select it within the right column corresponding to T/Task. The Preprocessor part is explained in the advanced section. If it is a rule file, select first the rule tab (see **ref to the picture**) and then select the desired rule file. Note that upon selection of a rule file, the name of the file is included in the command line and automatically include the required '-r' flag.
+
+4. **Priority**: Assign a priority number to the task. The expected value has to be an integer. Agents will be assigned to tasks in decreasing order of priority. A task with a priority 0 will not be processed even if agents are available. Default value is 0.
```

Comment:
priority 0 will mean that agents will not be assigned automatically to this task. but if you assign agents to it (and there is not higher priority work around), it will still process. so the third sentence here is maybe not 100% correct.